config/routes.rb

```diff
@@ -19,5 +19,9 @@
     end
   end
 
+  namespace :api do
+    resources :service_providers, only: [:index]
```

Comment:
I'm really enjoying how explicit we are with the paths (makes searching way easier)

But if the project already only does terse rails style, I'm fine with that. Do we want to enforce any consistency between idp and dashboard?

FWIW I prefer the explicit style, but this is not a big deal to me right now, I'd rather focus on shipping the  functionality.